### DISCUSSION TITLE
Limit in the amount of items in an in clause in a findmany query?

### DISCUSSION THREAD

**Discussion by rasberrylasagna** [prisma/prisma#7125] [Q&A] [answered]
Hey there, I was wondering if there was a limit to the amount of items in the in clause that I couldn't find.

I'm seeing some very strange behavior when the list is moderately large (around 15000+) entries.

Is this a known / expected limitation?

```
        where: {
          // id: {
          //   in: ["1", "2", ...],
          // },
        },
```

**Comment by ryands17** [comment, marked as answer]
@rasberrylasagna 👋 

It would be great if you could open an issue [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) with the reproduction steps so that we can look into this and see what limits we are hitting.
